### PR TITLE
change auth settings to affect entire site

### PR DESCRIPTION
**Pull request by tim-mcarthur-moj**
Image version tested: 1.0.160-dev

### CODE REVIEW

**Comment by tim-mcarthur-moj** [comment]
This is currently deployed in the aws-develop environment

File: docker/confd/templates/opg-lpa-front2.conf.tmpl [hunk 1]
```diff
@@ -25,7 +25,10 @@
         rewrite	^/assets/([0-9]+)/(.*) /assets/$2  break;
 
     }
-
+{{if exists "/opg/lpa/front/nginx/auth"}}
```
Comment (carinadigital) [inline docker/confd/templates/opg-lpa-front2.conf.tmpl]: Why at this particular position in the file? why not at the top?

Comment (tim-mcarthur-moj) [inline docker/confd/templates/opg-lpa-front2.conf.tmpl]: no particular reason. Just making sure it was outside the block that catches only .php files

Comment (carinadigital) [inline docker/confd/templates/opg-lpa-front2.conf.tmpl]: Could we have it before the `asstets` location block? I think it will cause less confusion

Comment (tim-mcarthur-moj) [inline docker/confd/templates/opg-lpa-front2.conf.tmpl]: can do

**Pull request merged**